Kentor.AuthServices/SAML2P/Saml2NameIdPolicy.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Kentor.AuthServices.Saml2P
+{
+    /// <summary>
+    /// NameIDPolicy
+    /// </summary>
+    public class Saml2NameIdPolicy
+    {
+        /// <summary>
+        /// Allow Create
+        /// </summary>
+        public bool AllowCreate { get; set; }
+        /// <summary>
+        /// Format
+        /// </summary>
+        public string Format { get; set; }

```

Comment:
The Format is a URI according to the SAML spec, would it make sense to have a URI instead of a string here?